### PR TITLE
Reverts antagonist changes, fixes #217 & #189

### LINKED ISSUE #217: Rev is broken
Since antag picker update, rev will never happen. If you try to force it, it says unable to start game mode, resseting lobby.

### PR DESCRIPTION
**Pull request by ghost**
This fixes #217 & #189

### CODE REVIEW

File: code/game/gamemodes/traitor/traitor.dm [hunk 1]
```diff
@@ -292,6 +292,8 @@
 
 	// find a radio! toolbox(es), backpack, belt, headset
 	var/loc = ""
+	world.log << "Found [joined_player_list.len] players in-game when equipping traitor [traitor_mob]."
```
Comment (nalarmg) [inline code/game/gamemodes/traitor/traitor.dm]: Debug code announcing to the entire server. Needs to be removed or world.log << replaced with Debug()

**Comment by Nokington** [comment]
Take reverting this https://github.com/HippieStationCode/HippieStation13/pull/231 out and I will be fine with this being merged. Also changelog maybe?

**Comment by ghost** [comment]
I'll push the changelog once this is merged, since there is a lot more that needs a changelog, also reverting #231 was unintentional, however we're adding sounds to antagonist jobs, so i think #231 will be obsolete by then (aka, later tonight)

**Comment by Nokington** [comment]
Ehhh, it is still good to have though because people probably won't know what the sounds mean.

**Comment by ghost** [comment]
there.

**Comment by nalarmg** [comment]
Code looks fine. If the headmins clear it, it's fine to be merged.

**Pull request merged**